customization/dark-theme.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: doc
+title: Dark Theme in Dom0 and DomU
+permalink: /doc/dark-theme/
+---
+
+Dark Theme in Dom0
+==================
+
+Dark KDE in Dom0
+----------------
+
+The following text describes how to change the default light theme to a dark theme. This is just an example, feel free to adjust the appearance to your taste.
+
+The image below shows the default light theme after installation.
+![begin light theme](/attachment/wiki/Dark-Theme/kde-fresh-installed-standard.png)
+
+This is the result after applying the steps described here.
+![end result dark theme](/attachment/wiki/Dark-Theme/kde-end-result.png)
+
+1 - Change `Workspace Appearance`
+
+  1. Open the `Workspace Appearance` window
```

Comment:
Putting spaces in front of the numbers in a numbered list will break the numbering.